### DISCUSSION TITLE
Keybinds for speccific sounds

### DISCUSSION THREAD

**Discussion by 7ArteeZy7** [MrBean355/admiralbulldog-sounds#49] [Ideas]
I know its been a while since this project was made, but a good feature would be being able to set up specific keybinds for a specific sound clip.
Also since to access the soundboard, we have to tab out of the game, another good idea would be being able to play soundclips with commands from discord(since discord provides an overlay with shift + ~), so we dont have to constantly tab out and tab in to use the soundboard

**Comment by MrBean355** [comment]
Hey @7ArteeZy7 
Thanks for the suggestions. Sorry for only getting back to you now.
Unfortunately it looks like adding hotkeys isn't possible, since it's a Java application.
So you'd like a Discord command like `!playsound kekw` to manually play sounds through Discord?

**Reply by 7ArteeZy7** [reply]
Yes, that would be the best alternative to hotkeys, discord gives an ingame overlay so its much better to be able to use playsounds through commands instead of tabbing out of the game. Although, this might be just my perspective and others might just prefer tabbing in and out to use a soundboard. Again, this is just a suggestion/idea. Love this project and hope you're doing great.

**Reply by MrBean355** [reply]
Thank you, will definitely take a look at implementing it!
